.github/workflows/create_pr_with_version_update.yaml

```diff
@@ -0,0 +1,91 @@
+name: Create PR on eCommerce project with version update
+
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+  pull_request:
+
+permissions:
+  pull-requests: write
+
+jobs:
+  create_pr:
+    strategy:
+      matrix:
+        repo:
+          - pagopa-ecommerce-transactions-service
+          - pagopa-ecommerce-scheduler-service
+
+    name: Create PR on ${{ matrix.repo }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout pagopa/${{ matrix.repo }}
+        uses: actions/checkout@v3
+        with:
+          repository: pagopa/${{ matrix.repo }}
+
+      - name: Get latest tag
+        id: latest_tag
+        uses: WyriHaximus/github-action-get-previous-tag@v1
+
+      - name: Replace old version with new version
+        uses: jacobtomlinson/gha-find-replace@v3
+        with:
+          find: "<pagopa-ecommerce-commons.version>\\d+\\.\\d+\\.\\d+</pagopa-ecommerce-commons.version>"
+          replace: "<pagopa-ecommerce-commons.version>${{ steps.latest_tag.outputs.tag }}</pagopa-ecommerce-commons.version>"
+          regex: true
+          include: "pom.xml"
+
```

Comment:
Duplicate this task in order to perform eCommerce commons version replacement also for helpdesk gradle build file. See https://github.com/pagopa/pagopa-ecommerce-helpdesk-service/blob/main/build.gradle.kts#L23

Reply:
Fixed in efe9936